Cartridges/int_checkoutcom/cartridge/scripts/helpers/eventsHelper.js

```diff
@@ -110,6 +120,7 @@ var eventsHelper = {
         paymentInstrument.paymentTransaction.setType(PaymentTransaction.TYPE_CAPTURE);
 
         setPaymentStatus(order);
+        order.setConfirmationStatus(2);
```

Comment:
What does this number mean ? Could you use a constant which indicate some meaningful words